specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/mfe.json

```diff
@@ -9547,6 +9556,23 @@
       "x-ms-discriminator-value": "AzureDataLakeGen2",
       "additionalProperties": false
     },
+    "AzureDatastore": {
+      "description": "Base definition for Azure datastore contents configuration.",
+      "type": "object",
+      "properties": {
+        "resourceGroup": {
+          "description": "Azure Resource Group name",
+          "type": "string",
+          "x-nullable": true
+        },
+        "subscriptionId": {
+          "description": "Azure Subscription Id",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "additionalProperties": false
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=7&length=20)

nit - ` "additionalProperties": false` seems redundant, default should be false.